<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

	<!-- Constants -->
	<xacro:property name="PI" value="3.1415926535897931"/>

	<!-- baseproperties -->
	<xacro:property name="baseLength" value="0.0711632"/>
	<xacro:property name="baseRadius" value="0.16495"/>
	<xacro:property name="baseMass" value="2"/>

	<!-- Caster wheel properties -->
	<xacro:property name="casterRadius" value="0.018"/>
	<xacro:property name="casterMass" value="0.01"/>
	<xacro:property name="casterOffsetX" value="0.12"/>
	<xacro:property name="casterOffsetZ" value="0.001"/>

	<!-- Differential-drive wheels properties -->
	<xacro:property name="wheelWidth" value="0.023"/>
	<xacro:property name="wheelWidthforCollision" value="0.001"/>
	<xacro:property name="wheelRadius" value="0.033"/>
	<xacro:property name="wheelMass" value="0.01"/>
	<xacro:property name="wheelOffsetY" value="0.12"/>
	<xacro:property name="wheelOffsetZ" value="0.0151"/>

	<!-- Hokuyo laser properties -->
	<xacro:property name="hokuyoMass" value="0.00"/> 
	<xacro:property name="hokuyoLength" value="0.05"/>
	<xacro:property name="hokuyoWidth" value="0.05"/>
	<xacro:property name="hokuyoHeight" value="0.07"/>
	<xacro:property name="hokuyoOffsetX" value="0.1"/>
	<xacro:property name="hokuyoOffsetZ" value="0.1335"/>

	<!-- Camera properties -->
	<xacro:property name="cameraLength" value="0.04"/>
	<xacro:property name="cameraWidth" value="0.08"/>
	<xacro:property name="cameraHeight" value="0.03"/>
	<xacro:property name="cameraMass" value="0.00"/>
	<xacro:property name="cameraOffsetX" value="0.165"/>
	<xacro:property name="cameraOffsetZ" value="0.06"/>

</robot>
